models/engine/db_storage.py

```diff
@@ -74,3 +74,23 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve one object based on class and ID,
+        or None if not found"""
+
+        if type(cls) is str:
```

Comment:
Would put this data check in its own function... it seems pretty useful that other objects/functions would want to use.